utopia-vscode-extension/snippets.json

```diff
@@ -30,282 +22,140 @@
-      "},"
-    ],
-    "description": "Register Module Expression Control"
+  "Vector4 Prop Control": {
+    "prefix": "Utopia.vector4Control()",
+    "body": "Utopia.vector4Control()"
   },
-  "registerModule:array property control": {
-    "prefix": "registerModule:Array Control",
+  "Array Prop Control": {
+    "prefix": "Utopia.arrayControl()",
     "body": [
-      "${1:PropName}: {",
-      "  control: 'array',",
-      "  label: '${1:PropName}'",
-      "  //visibleByDefault: true",
-      "  //defaultValue: ['A', 'B']",
-      "  propertyControl: {control: 'string-input'}",
-      "  //maxCount: 5",
-      "},"
-    ],
-    "description": "Register Module Array Control"
+      "Utopia.arrayControl(",
+      "  ${1:Utopia.stringControl()} // control to use for each value in the array",
+      ")"
+    ]
   },
-  "registerModule:object property control": {
-    "prefix": "registerModule:Object Control",
+  "Fixed Size Array Prop Control": {
+    "prefix": "Utopia.fixedSizeArrayControl()",
     "body": [
-      "${1:PropName}: {",
-      "  control: 'object',",
-      "  label: '${1:PropName}'",
-      "  //visibleByDefault: true",
-      "  //defaultValue: {itemID: 10, name: 'Hat'}",
-      "  object: { itemID: {label: 'itemID', control: 'number-input'}, name: {label: 'name', control: 'string-input'} }",
-      "},"
-    ],
-    "description": "Register Module Array Control"
+      "Utopia.fixedSizeArrayControl(",
+      "  ${1:Utopia.stringControl()}, // control to use for each value in the array",
+      "  ${2:10 /* length */}",
+      ")"
+    ]
   },
-  "registerModule:union property control": {
-    "prefix": "registerModule:Union Control",
-    "body": [
-      "${1:PropName}: {",
-      "  control: 'union',",
-      "  label: '${1:PropName}'",
-      "  //visibleByDefault: true",
-      "  //defaultValue: 'Hat'",
-      "  controls: [{control: 'string-input'}, {control: 'number-input'}]",
-      "},"
-    ],
-    "description": "Register Module Union Control"
+  "Object Prop Control": {
+    "prefix": "Utopia.objectControl()",
+    "body": ["Utopia.objectControl({", "  ${1:key}: ${2:Utopia.stringControl()},", "})"]
   },
-  "registerModule:tuple property control": {
-    "prefix": "registerModule:Tuple Control",
+  "Tuple Prop Control": {
+    "prefix": "Utopia.tupleControl()",
     "body": [
-      "${1:PropName}: {",
-      "  control: 'tuple',",
-      "  label: '${1:PropName}'",
-      "  //visibleByDefault: true",
-      "  //defaultValue: ['Hat', 9]",
-      "  propertyControls: [{control: 'string-input'}, {control: 'number-input'}]",
-      "},"
-    ],
-    "description": "Register Module Tuple Control"
+      "Utopia.tupleControl([",
+      "  ${1:Utopia.stringControl()},",
+      "  ${2:Utopia.numberControl()},",
+      "])"
+    ]
   },
-  "registerModule:folder property control": {
-    "prefix": "registerModule:Folder Control",
+  "Union Prop Control": {
```

Comment:
I think this naming will mean users wont know how to find it, especially with the shorter `Prop` in the name.

Reply:
I'll update `Prop` to `Property`

Reply:
Done